app/models/miq_report/generator/trend.rb

```diff
@@ -148,10 +148,10 @@ def calc_value_at_target(limit, trend_data_key, trend_data)
           if Time.at(result).utc <= Time.now.utc
             return Time.at(result).utc.strftime("%m/%d/%Y")
           else
-            return "#{((Time.at(result).utc - Time.now.utc) / 1.day).round} days, on #{Time.at(result).utc.strftime("%m/%d/%Y")} (#{get_time_zone("UTC")})"
+            return _("#{((Time.at(result).utc - Time.now.utc) / 1.day).round} days, on #{Time.at(result).utc.strftime("%m/%d/%Y")} (#{get_time_zone("UTC")})")
```

Comment:
This is dynamically generated string and simply applying `_()` will not work.

Reply:
fixed